src/controller/eme-controller.js

```diff
@@ -279,7 +281,10 @@ class EMEController extends EventHandler {
 
     this.hls.startLoad();
     keySession.addEventListener('message', (event) => {
-      this._onKeySessionMessage(keySession, event.message);
```

Comment:
`keySession === event.target`, keep this aligned with https://github.com/Tubitv/hls.js/pull/5/files#diff-66dbb17319c4f0d5f009baca4e3ecf36R287